pkgs/development/compilers/rust/rustc.nix

```diff
@@ -107,6 +107,8 @@ in stdenv.mkDerivation rec {
     "${setHost}.musl-root=${pkgsBuildHost.targetPackages.stdenv.cc.libc}"
   ] ++ optionals stdenv.targetPlatform.isMusl [
     "${setTarget}.musl-root=${pkgsBuildTarget.targetPackages.stdenv.cc.libc}"
+  ] ++ optionals (rust.toRustTargetNoStd stdenv.targetPlatform) [
```

Comment:
```suggestion
  ] ++ optionals (rust.IsNoStdTarget stdenv.targetPlatform) [
```

Reply:
Made these changes, thanks for the review!